src/pages/home/report/ReportAttachments.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import AttachmentModal from '../../../components/AttachmentModal';
+import Navigation from '../../../libs/Navigation/Navigation';
+import * as ReportUtils from '../../../libs/ReportUtils';
+
+const propTypes = {
+    /** Navigation route context info provided by react navigation */
+    route: PropTypes.shape({
+        /** Route specific parameters used on this screen */
+        params: PropTypes.shape({
+            /** The report ID which the attachment is associated with */
+            reportID: PropTypes.string.isRequired,
+            /** The uri encoded source of the attachment */
+            source: PropTypes.string.isRequired,
+        }).isRequired,
+    }).isRequired,
+};
+
+function ReportAttachments(props) {
+    const reportID = _.get(props, ['route', 'params', 'reportID']);
+    const report = ReportUtils.getReport(reportID);
+    const source = decodeURI(_.get(props, ['route', 'params', 'source']));
```

Comment:
The `source` can be a number (viewing assets in native) but the decoded uri is always a string which makes the image fail to load. We needed to preserve the type of the passed source. (Coming from https://github.com/Expensify/App/issues/31138)